app/_hub/kong-inc/aws-lambda/index.md

```diff
@@ -203,29 +208,39 @@ from being executed. We are planning to remove this limitation in the future.
+with lambda functions, and create users and roles.
+
+1. Create an Execution role in AWS.
+2. Create a user that will invoke the function via Kong and test it.
+3. Create a Service and Route in Kong, add the aws-lambda plugin linked to
+our AWS function, and execute it.
 
 #### Configure
 
-1. First, let's create an execution role called `LambdaExecutor` for our lambda function.
+1. First, let's create an execution role called `LambdaExecutor` for our
+lambda function.
 
-    In IAM Console create a new Role choosing the AWS Lambda service, there will be no policies as our function in this example will simply execute itself giving us back an hardcoded JSON as response without accessing other AWS resources.
+    In the IAM Console, create a new Role choosing the AWS Lambda service. There
+    will be no policies as our function in this example will simply execute
+    itself, giving us back a hardcoded JSON response without accessing other
+    AWS resources.
 
-2. Now let's create a user named KongInvoker, used by our Kong API gateway to invoke the function.
+2. Now let's create a user named `KongInvoker`, used by our Kong API gateway
+to invoke the function.
 
-    In IAM Console create a new user, must be provided to it programmatic access via Access and Secret keys; then will attach existing policies directly particularly the AWSLambdaRole predefined. Once the user creation is confirmed, store Access Key and Secret Key in a safe place.
+    In the IAM Console, create a new user. Programmatic access must be provided to the user via Access and Secret keys. Then attach existing policies directly, particularly the predefined `AWSLambdaRole`. After the user creation is confirmed, store the Access Key and Secret Key in a safe place.
 
-3. Now we need to create the lambda function itself, will do so in N.Virginia Region (code us-east-1).
+3. Now we need to create the lambda function itself; we will do so in
+N. Virginia Region (code `us-east-1`).
 
-    In Lambda Management, create a new function Mylambda, there will be no blueprint as we are going to paste the code below (which is an example code snippet); for the execution role let's choose an existing role specifically LambdaExecutor created previously.
+    In Lambda Management, create a new function `Mylambda`. There will be no blueprint because we are going to paste the code below (which is an example code snippet). Let's choose an existing role for the execution role, specifically `LambdaExecutor` created previously.
 
-    **Note**: The following code snippet is only an example. The Kong AWS Lambda plugin supports all runtimes provided by AWS. See the list of runtimes in the **AWS Lambda** > **Functions** > **Create function** dialog. 
+    **Note**: The following code snippet is only an example. The Kong AWS Lambda plugin supports all runtimes provided by AWS. See the list of runtimes in the **AWS Lambda** > **Functions** > **Create function** dialog.
```

Comment:
```suggestion
    >**Note**: The following code snippet is only an example. The Kong AWS Lambda plugin supports all runtimes provided by AWS. See the list of runtimes in the **AWS Lambda** > **Functions** > **Create function** dialog.
```

Reply:
I think the slanted text produced by > is more difficult to read.